neo4j-cypher-dsl/src/test/java/org/neo4j/cypherdsl/core/CypherIT.java

```diff
@@ -4056,32 +4073,49 @@ void prettyPrintingWithDefaultSettingShouldWork() {
 			assertThat(Renderer.getRenderer(Configuration.prettyPrinting()).render(statement))
 				.isEqualTo(
 					"MATCH (u:User)\n" +
-					"WHERE (u.name = 'Max'\n" +
-					"  AND u.lastName = 'Mustermann'\n" +
-					"  AND EXISTS {\n" +
-					"    MATCH (u)-[:LIKES]->(b:Bike)\n" +
-					"    WHERE ((b)-[:LINK]->()\n" +
-					"      OR EXISTS {\n" +
-					"        MATCH (b)-[:LINK]->()\n" +
-					"      })\n" +
-					"  })\n" +
-					"SET u.lastName = $newName\n" +
-					"WITH u\n" +
-					"MATCH (b:Bike)\n" +
-					"CREATE (u)-[:LIKES]->(b)\n" +
-					"RETURN u {\n" +
-					"  name: u.name,\n" +
-					"  nesting1:  {\n" +
-					"    name: u.name,\n" +
-					"    nesting2:  {\n" +
-					"      name: b.name,\n" +
-					"      pattern: [(u)-[:LIKES]->(other) WHERE other.foo = $foo | other {\n" +
-					"        .x,\n" +
-					"        .y\n" +
-					"      }]\n" +
-					"    }\n" +
-					"  }\n" +
-					"}"
+							"WHERE (u.name = 'Max'\n" +
+							"  AND u.lastName = 'Mustermann'\n" +
+							"  AND EXISTS {\n" +
+							"    MATCH (u)-[:LIKES]->(b:Bike)\n" +
+							"    WHERE ((b)-[:LINK]->()\n" +
+							"      OR EXISTS {\n" +
+							"        MATCH (b)-[:LINK]->()\n" +
+							"      })\n" +
+							"  })\n" +
+							"SET u.lastName = $newName\n" +
+							"WITH u\n" +
+							"MATCH (b:Bike)\n" +
+							"CREATE (u)-[:LIKES]->(b)\n" +
+							"WITH u\n" +
+							"CALL {\n" +
+							"  WITH u\n" +
+							"  MATCH (u)-[:SOMETHING]->(x)\n" +
+							"  CALL {\n" +
+							"    WITH x\n" +
+							"    MATCH (x)-[:DEEPER]->(y)\n" +
+							"    RETURN y {\n" +
+							"      .bar\n" +
+							"    } AS bar\n" +
+							"  }\n" +
+							"  RETURN x {\n" +
+							"    .foo,\n" +
+							"    bar: bar\n" +
+							"  } AS anyThing\n" +
+							"}\n" +
+							"RETURN u {\n" +
+							"  name: u.name,\n" +
+							"  anyThing: anyThing,\n" +
+							"  nesting1:  {\n" +
+							"    name: u.name,\n" +
+							"    nesting2:  {\n" +
+							"      name: b.name,\n" +
+							"      pattern: [(u:User)-[:LIKES]->(other) WHERE other.foo = $foo | other {\n" +
```

Comment:
@michael-simons I guess having the label here at `u` (User) is a bug

Reply:
Yes indeed.